scripts/scil_compute_metrics_stats_in_ROI.py

```diff
@@ -104,20 +122,56 @@ def main():
         metrics_files = [nib.load(f) for f in args.metrics_file_list]
 
     # Compute the mean values and standard deviations
-    stats = get_roi_metrics_mean_std(mask_data, metrics_files)
-
-    roi_name = split_name_with_nii(os.path.basename(args.in_mask))[0]
-    json_stats = {roi_name: {}}
-    for metric_file, (mean, std) in zip(metrics_files, stats):
-        metric_name = split_name_with_nii(
-            os.path.basename(metric_file.get_filename()))[0]
-        json_stats[roi_name][metric_name] = {
-            'mean': mean.item(),
-            'std': std.item()
-        }
+    nb_mask = len(mask_list)
+    nb_metric = len(metrics_files)
+    stats_list = []
+    for i in range(nb_mask):
```

Comment:
this looks like L135, don't need to loop twice.

Reply:
Should have a way to have a cleaner code here

Reply:
I tried to make minimal changes, now it should be better